src/main/java/org/jasig/portlet/widget/links/ResourceLinksController.java

```diff
@@ -0,0 +1,49 @@
+package org.jasig.portlet.widget.links;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import javax.portlet.PortletRequest;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.portlet.bind.annotation.RenderMapping;
+
+@Controller
+@RequestMapping("VIEW")
+public class ResourceLinksController extends ResourceLinksBaseController {
+
+    private final Logger log = LoggerFactory.getLogger(this.getClass());
```

Comment:
📓 Lombok provides a nice helper to normalize log usage
Ref: https://projectlombok.org/features/log